src/include/user/abi_dbg.h

```diff
@@ -19,7 +19,7 @@
 #define __USER_ABI_DBG_H__
 
 #define SOF_ABI_DBG_MAJOR 5
-#define SOF_ABI_DBG_MINOR 0
+#define SOF_ABI_DBG_MINOR 1
```

Comment:
Kernel is not translating UUIDs, it passing them directly for topology loading.

Reply:
In proposed solution, it is as you wrote. Here logger is responsible for translation UUID value to UUID key/address value.

Reply:
The commit message is saying the kernel will do this (and there is no way to comment on the commit message).